src/test/java/com/google/plantasy/TimingPropertiesTest.java

```diff
@@ -50,61 +50,76 @@ public void getQuizTimestampProperty_nullUserTimeStamp() {
         Entity user = new Entity("user");
         user.setProperty("quiz_timestamp", null);
         user.setProperty("userID", "abc");
- 
+
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         datastore.put(user);
- 
-        Long actual = timing_properties_test.getQuizTimestampProperty("user", "userID", (user.getProperty("userID")).toString(), datastore); 
-        Assert.assertEquals(null, actual);
+
+        Long actual = null;
+        try {
+            actual = timing_properties_test.getQuizTimestampProperty("user", "userID", (user.getProperty("userID")).toString(), datastore); 
+        } catch (NullPointerException e) {
```

Comment:
I don't think you're testing what you intend to test here. When the `quiz_timestamp` in datastore is `null`, it makes sense for `getQuizTimestampProperty` to return `null`, right? Because `null` is the correct answer to "What is timestamp for this quiz?"

So there's no need for the try-catch block here. If this input causes a `NullPointerException`, the test ought to fail.

Reply:
Yes I took this out! This is why some tests will still assert nulls while the others have exception checks